packages/toolpad-studio/src/server/toolpadAppBuilder.ts

```diff
@@ -26,11 +27,10 @@ function getHtmlContent(entry: string) {
     <html lang="en">
       <head>
         <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
         <title>Toolpad</title>
-        <link
-          rel="stylesheet"
-          href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
-        />
+        <link rel="preload" href="${FONTS_URL}" as="style" onload="this.onload=null;this.rel='stylesheet'">
```

Comment:
Sorry for not reviewing earlier!
All looks good, not sure if these would also help with font loading performance:
```
<link rel="preconnect" href="https://fonts.googleapis.com" />
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Reply:
I don't think it would make a difference, the browser will already be loading the css asap.